src/workers/personalizedDigestEmail.ts

```diff
@@ -166,6 +192,55 @@ const worker: Worker = {
       return;
     }
 
+    const analyticsQueue = fastq.promise(
+      async (
+        data: {
+          user_id: string;
+          device_id: string;
+          experiment_id: string;
+          variation_id: number;
+        } & AnalyticsEvent,
+      ) => {
+        await sendAnalyticsEvent([data]);
+      },
+      1,
+    );
+
+    const growthbookClient = getUserGrowthBookInstace(user.id, {
+      enableDevMode: process.env.NODE_ENV !== 'production',
+      subscribeToChanges: false,
+      trackingCallback: async (experiment, result) => {
```

Comment:
* I think we should have the `trackingCallback` support in `getUserGrowthBookInstace`. This will be very useful for future development. For your consideration though
* We don't need the queue here imo, we just send the request because each user is a single request. Again for your consideration
* The request you send here is actually the wrong one. We don't need analytics event, we need experiment allocation. It's the request that goes to `/e/x`. It doesn't require `event_name`. For device_id, we better just generate uuid

Reply:
As for analytics event endpoint/payload I updated in 7d90a0a93331df435d77cf7a9b301180c5fbcbd1